docs/snippets/quickstart-install.md

```diff
@@ -1,22 +1,59 @@
+!!! todo "Installing the `quickstart` plugin"
+    === "Using Homebrew"
+        For macOS, you can install the `quickstart` plugin by using <a href="https://brew.sh" target="_blank">Homebrew</a>.
+            ```
+            brew install knative-sandbox/kn-plugins/quickstart
+            ```
 
+    === "Using a binary"
+         You can install the `quickstart` plugin by downloading the executable binary for your system and placing it on your `PATH` (for example, in `/usr/local/bin`).
 
-`konk` is a shell script that completes the following functions:
+         A link to the latest stable binary release is available on the <a href="https://github.com/knative-sandbox/kn-plugin-quickstart/releases"> `quickstart` release page</a>.
+
+    === "Using Go"
+        1. Check out the `kn-plugin-quickstart` repository:
+
+              ```
+              git clone https://github.com/knative-sandbox/kn-plugin-quickstart.git
+              cd kn-plugin-quickstart/
+              ```
+
+        1. Build an executable binary:
+
+              ```
+              hack/build.sh
+              ```
+
+        1. Move the executable binary file to the `kn` plugins directory:
```

Comment:
I would not preferred to advertise to end users the use of this config directory, I preferred to advertise that the plugin needs be in the `$PATH`.

You can tell the user to include the current directory in `PATH`
```
PATH=$PATH:$PWD
```
Or move the file to an existing directory in the `PATH` like
```
mv kn-quickstart $GOPATH/bin/
```

I think using `$PWD` is better option if they are contributing or changing the go source code, they can compile again with the change and re-run `kn quickstart` using the plugin from the current directory.

Reply:
I _think_ I prefer the install to existing PATH directory option because (1) it's slightly easier, and (2) contributors to the plugin probably aren't using the quickstart docs to get set up. I updated to use that option, but am happy to change if that's the better option.